### PR TITLE
Backported PR #2962 to 2.8

### LINKED ISSUE #2948: 2.8: webui: incorrect download url when api_prefix is used
Issue/Feature description
====================

Hello,

I am running private obs instance, my ```options.yml``` is the following:

```
frontend_host: mydomain
frontend_port: 443
frontend_protocol: https
api_relative_url_root: /obs
```

Expected result
============
When I open pages at https://mydomain/obs/package/binary/project/package
(```app/views/webui/package/binary.html.erb```)
I see text "Detailed Information About foo.rpm" with wrong url attached to foo.rpm link.

It references to ```https://mydomain/build/project/....``` whereas it expected to be ```https://mydomain/obs/build/project/....```

### PR DESCRIPTION
**Pull request by mdeniz**
The PR #2962 that fixed #2948 is now in 2.8 release branch.

### CODE REVIEW

**Comment by adrianschroeter** [comment]
maybe add some "obsolete" before the sentence in the ReleaseNotes file to give some reason for this.

**Comment by adrianschroeter** [comment]
and this should be documented in the ReleaseNotes-2.8.1 file, not 2.8

**Comment by mdeniz** [comment]
No ReleaseNotes-2.8.1 file exists already... so, should I create it?

**Comment by mdeniz** [comment]
@adrianschroeter  should I move the ReleaseNotes-2.8 to the `docs` subdir?

**Comment by mdeniz** [comment]
@adrianschroeter  done, please re-review 👍

File: ReleaseNotes-2.8.1 [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#
+# openSUSE Build Service 2.8.1
+#
+
+Updaters from OBS 2.8.0 release can just ugrade the packages
```
Comment (adrianschroeter) [inline ReleaseNotes-2.8.1]: upgrade

File: ReleaseNotes-2.8.1 [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+and restart all services. Updaters from former releases should
+read the README.UPDATERS file.
+
+This is in first place a bugfix release focusing on security issues
```
Comment (adrianschroeter) [inline ReleaseNotes-2.8.1]: We don't have security issues fixed so far

File: ReleaseNotes-2.8.1 (hunk 1 above)
Comment (mdeniz) [inline ReleaseNotes-2.8.1]: Done

File: ReleaseNotes-2.8.1 (hunk 2 above)
Comment (mdeniz) [inline ReleaseNotes-2.8.1]: Done

**Pull request merged**